test-case-reporting/test/test_result_record.test.ts

```diff
@@ -127,25 +139,60 @@ describe('TestResultRecord', () => {
         'test_cases'
       ).select();
 
-      const sampleTestCases: Array<DB.TestCase> = sampleKarmaReport.browsers.results.map(
-        result => {
-          const {suite, description} = result;
-          const name = testResultRecord.makeTestCaseName(suite, description);
-          return {
-            id: md5(name),
-            name,
-          };
-        }
-      );
+      const sampleTestCases: Array<DB.TestCase> = [
+        {
+          id: '8a3d71d66b2913bb981a8d4f2a2930db',
+          name: 'when test is bad | it fails',
+        },
+        {
+          id: 'c5cf7c15d50ec660c3b10b6c91bfe3f8',
+          name: 'when test was skipped | it skipped',
+        },
+        {
+          id: '36340965686c32694f88f06c6a3f71ac',
+          name: '🤖 when passing test has emojis 🤖 | it passes 🎉',
+        },
+      ];
```

Comment:
Nit: Typically you'd write `expect(resultFromTestedFn).toMatchObject(hardCodedExpectation)`. Effectively the same, but the output "Expected X to match Y, but got Z" will be swapped around otherwise.

Reply:
When doing `.toMatchObject`, it matches the first object to a subset of the properties of the second one, so it won't work in the reverse order -- I'll see if I can find something that reverses the order.

Reply:
Working on this, will let you know when done.

Reply:
Done.